### DISCUSSION TITLE
Ignore invisible/see-through objects during building

### DISCUSSION THREAD

**Discussion by PhyxionNL** [OpenRCT2/OpenRCT2#15661] [Feature requests] [answered]
For example, you have the rides and supports invisible/see-through. When you want to delete a path, it will still select the invisible/see-through objects (Can't start construction on ...) but what I expect is the selection to ignore those objects and delete (or build) the path I have my cursor over instead.

**Comment by Nubbie** [comment]
Cut-away view partially solves this

**Comment by J-PIE-314** [comment, marked as answer]
This has been implemented in #22217.
